models/public/yolo-v1-tiny-tf/yolo-v1-tiny-tf.md

```diff
@@ -89,10 +89,10 @@ The array of detection summary info, name - `conv2d_9/BiasAdd`,  shape - `1,13,1
 - `Cx`, `Cy` - cell index
 
 Detection box has format [`x`,`y`,`h`,`w`,`conf`,`class_no_1`, ..., `class_no_20`], where:
-- (`x`,`y`) - coordinates of box center, relative to cell
-- `h`,`w` - normalized height and width of box
-- `conf` - confidence of detection box
-- `class_no_1`,...,`class_no_20` - score for each class in logits format
+- (`x`,`y`) - raw coordinates of box center, apply [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to get coordinates relative to the cell 
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function) and multiply by corresponding anchors to get height and width values relative to cell
```

Comment:
Maybe you could specify the base for the exponential function too? Otherwise, there's not enough detail to decode the output.

Reply:
Well, usually, if *exponential function* specified without base it is natural by default. If I could add formula to markdown, I'll prefer this instead of letter "e" - it's look messy.

Reply:
Yeah, OK.